### PR TITLE
frontend/#2031/ bundle optimization

### PR DESCRIPTION
**Pull request by ashishsingh101**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [x] Dependency updates

## Description

1. increased webpack from v4 to v5 and its dependent dependencies versions as well to keep updated with latest features, also this reduces zipped bundle size by ~8KB

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Frontend/ui-customer/src/Flow.purs [hunk 1]
```diff
@@ -309,20 +309,20 @@ currentFlowStatus = do
...
-  _ <- lift $ lift $ liftFlow $ logEvent logField_ "ny_user_choose_lang_scn_view"
-  flow <- UI.chooseLanguageScreen
-  case flow of
-    NextScreen language -> do
-                            setValueToLocalStore LANGUAGE_KEY language
-                            void $ pure $ setCleverTapUserProp [{key : "Preferred Language", value : unsafeToForeign language}]
-                            _ <- lift $ lift $ liftFlow $(logEventWithParams logField_ "ny_user_lang_choose" "language" (language))
-                            enterMobileNumberScreenFlow
-    Refresh state -> chooseLanguageScreenFlow
+-- chooseLanguageScreenFlow :: FlowBT String Unit
```
Comment (witcher-shailesh) [inline Frontend/ui-customer/src/Flow.purs:312]: why is this commented? please add proper comments for it (if needed to be commented)

File: Frontend/ui-driver/src/Flow.purs [hunk 2]
```diff
@@ -289,14 +289,14 @@ enterOTPFlow = do
...
 
--- TODO :: As per the api response check the floe for driver registration
-registrationStatusFlow :: FlowBT String Unit
-registrationStatusFlow = do
-  let startTime = getTime unit
-  -- resp <- Remote.driverRegistrationStatus (DriverResgistrationStatusReq {})
-  let endTime = getTime unit
-  let registrationStatus = driverRegistrationStatus
-  if registrationStatus then currentRideFlow Nothing else permissionsScreenFlow Nothing Nothing
+-- TODO :: As per the api response check the flow for driver registration, will need in future
```
Comment (witcher-shailesh) [inline Frontend/ui-driver/src/Flow.purs:292]: what's the future need of this?

**Review by witcher-shailesh — COMMENTED** [review]
Is it tested with one APK generation? can we update the package JSON accordingly in bitbucket as well? As while generation of bundle that package.json is used

File: Frontend/ui-driver/src/Flow.purs (hunk 2 above)
Comment (ashishsingh101) [reply Frontend/ui-driver/src/Flow.purs:292]: just commented code which are defined but not being used

**Comment by ashishsingh101** [comment]
> Is it tested with one APK generation? can we update the package JSON accordingly in bitbucket as well? As while generation of bundle that package.json is used

yes it is tested, we can update in bitbucket after some while to make sure there is no other need as it will get tested by devs and qas

**Review by witcher-shailesh — APPROVED** [review]

**Pull request merged**